packages/extension-browser/src/devtools/views/controls/settings.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { FormEvent, useCallback } from 'react';
+
+import { getMessage } from '../../utils/i18n';
+
+import Summary from './summary';
+import ConfigLabel from '../pages/config/label';
+import Checkbox from './checkbox';
+import LabelText from './label-text';
+
+import * as styles from './settings.css';
+
+const settings = { telemetry: 'telemetry' };
+
+type Props = {
+    isTelemetryEnabled: boolean;
+    onTelemetryChange: (enable?: boolean) => void;
```

Comment:
Why is `enable` optional? Looks like we always pass `true` or `false`. Narrowing the type here should also simplify checks in components which consume this.

Reply:
I thought I will need that for "close" the notification, but that is not necessary and I forgot to change that :)